src/ibeo_core.cpp

```diff
@@ -41,357 +41,378 @@ const int32_t AS::Drivers::Ibeo::HostVehicleState2807::DATA_TYPE = 0x2807;
 const int32_t AS::Drivers::Ibeo::DeviceStatus::DATA_TYPE = 0x6301;
 
 // Primitive parsing functions
-void MountingPositionF::parse(uint8_t *in)
+void MountingPositionF::parse(const std::vector<uint8_t>& in, const uint16_t& offset)
```

Comment:
Good catch. I'll fix these.

Reply:
@Daniel-Stanek - It looks like the only ones where it wasn't being used were the ones where parse_tuple was called. I had to add an offset parameter to parse_tuple and then updated these to pass it along. The only exception is IbeoHeader::parse and, since that should always come at the beginning of a message, no offset should be necessary.